sqlalchemy-stubs/sql/sqltypes.pyi

```diff
@@ -203,26 +216,30 @@ class JSON(Indexable, TypeEngine[Dict[str, Any]]):
     class JSONPathType(JSONElementType): ...
     comparator_factory: Any = ...
     @property
-    def python_type(self): ...
+    def python_type(self) -> Type[dict]: ...
     @property
-    def should_evaluate_none(self): ...
+    def should_evaluate_none(self) -> bool: ...  # type: ignore  # incompatible with supertype
```

Comment:
It is always better to mention which supertype it is incompatible with (here and below).